frontend/src/component/admin/network/NetworkTrafficUsage/hooks/useStats.ts

```diff
@@ -0,0 +1,98 @@
+import { type ChartDataSelection, toDateRange } from '../chart-data-selection';
+import { useTrafficSearch } from 'hooks/api/getters/useInstanceTrafficMetrics/useInstanceTrafficMetrics';
+import { currentDate } from '../dates';
+import { useMemo } from 'react';
+import {
+    toTrafficUsageChartData as newToChartData,
+    toConnectionChartData,
+} from '../chart-functions';
+import {
+    calculateEstimatedMonthlyCost,
+    calculateOverageCost,
+    calculateTotalUsage,
+} from 'utils/traffic-calculations';
+import { BILLING_TRAFFIC_BUNDLE_PRICE } from '../../../billing/BillingDashboard/BillingPlan/BillingPlan';
+import { averageTrafficPreviousMonths } from '../average-traffic-previous-months';
+
+export const useTrafficStats = (
```

Comment:
no changes, extracted component and split traffic stats and consumption stats calculation

Reply:
also didn't find tests for this code so will add them in the next PR